### ISSUE TITLE
Issues

### ISSUE THREAD

**Issue by geogos999** [aspnet/Docs#6341]
So I've followed the steps exactly and recreated them multiple times, and am still getting this error when clicking the "Movie App" button: 

InvalidOperationException: The layout view 'Welcome.cshtml' could not be located. The following locations were searched:
/Views/Movies/Welcome.cshtml
Microsoft.AspNetCore.Mvc.Razor.RazorView.GetLayoutPage(ViewContext context, string executingFilePath, string layoutPath)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1d975a83-1550-b284-d0d8-7fe2ea206ff5
* Version Independent ID: 48e81b42-7c2b-afbf-71e6-d4f442a4bd1f
* Content: [Add a model to an ASP.NET Core MVC app](https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-mvc-app/adding-model?view=aspnetcore-2.1)
* Content Source: [aspnetcore/tutorials/first-mvc-app/adding-model.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/tutorials/first-mvc-app/adding-model.md)
* Service: **asp.net-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
@geogos999 that's a really great error message. I can tell you the tutorial works. Let us know when you find your mistake.

**Comment by geogos999** [comment]
So I've figured out how to continue with the tutorial. I've needed to delete the Layout = "Welcome.cshtml" line from all of the cshtml pages under the Movies folder. Not sure if this is intentional for people to have to find, but honestly as a tutorial series trying to dive into web apps, i feel like this should have been stated in the tutorial on this page or the previous one. Unless I'm missing something on my end, that is. Hopefully this helps someone else that will inevitably have this issue.
